docker/Dockerfile

```diff
@@ -2,8 +2,11 @@
 FROM registry.access.redhat.com/rhel7:7.8-265
 
 RUN set -x && \
+    yum-config-manager --save --setopt=skip_missing_names_on_install=False && \
+    yum-config-manager --save --setopt=skip_missing_names_on_update=False && \
+    yum-config-manager --save --setopt=rhel-7-server-extras-rpms.skip_if_unavailable=true &&\
```

Comment:
Either `rhel-7-server-extras-rpms` is not needed and it can be removed from `--enablerepo`, or it should fail if unavailable

Reply:
fixed this @cvarjao. Thanks